deploy/stacks/ses_stack.py

```diff
@@ -0,0 +1,92 @@
+from aws_cdk import (
+    aws_ssm as ssm,
+    aws_kms as kms,
+    aws_sns as sns,
+    aws_ses as ses,
+    aws_route53 as route53,
+    aws_iam as iam,
+    RemovalPolicy,
+)
+
+
+from .pyNestedStack import pyNestedClass
+
+
+class SesStack(pyNestedClass):
+    def __init__(
+            self,
+            scope,
+            id,
+            envname='dev',
+            resource_prefix='dataall',
+            custom_domain=None,
```

Comment:
@TejasRGitHub, why don't we pass the email id here? Aren't we going to use it in the EmailIdentity creation as identity parameter?

Reply:
Hi @dlpzx , the email id is not used to create an email identity. We are only using the email id as a sender email id. The way currently SES stack is working , is that it is verifying the custom_domain and not the email id. When the custom_domain is verified any email id on that domain can be used to send an email. Also, verifying an email id could be possible in case the user has access to the inbox of that email id. Apart from that if an email id is verified then the user also has to make entries for SPF and DKIM into their DNS server (https://docs.aws.amazon.com/ses/latest/dg/email-authentication-methods.html) for the recipient to receive the email properly ( without it being sent to spam)